scripts/ci/check_compliance.py

```diff
@@ -1495,6 +1495,49 @@ def run(self):
                                       p.line, col=p.column, desc=p.desc)
 
 
+class SphinxLint(ComplianceTest):
+    """
+    SphinxLint
+    """
+
+    name = "SphinxLint"
```

Comment:
Can you add this to https://github.com/zephyrproject-rtos/zephyr/blob/main/.gitignore#L75? Honestly at this point it'd be cool to have these in a subdirectory or something and ignore the whole thing but let's add it for now

Reply:
Duh - thanks! I knew I saw that txt file earlier but thought I dreamt it or that it came from other tests I did with the CLI. Didn't realize it's actually an output from the script...

Reply:
done (and tested)